admin-ui/src/pages/requests-page.tsx

```
@@ -664,6 +669,19 @@ export function RequestsPage(): React.JSX.Element {
                           )}
                         >
                           {r.account_id || ""}
+                          {r.affinity_hit ? (
```

ESLint

[error] Delete `·(`

(prettier/prettier)